content/_includes/ClusterSetup.md

```diff
@@ -11,12 +11,11 @@ Follow this procedure for each TrueNAS SCALE system that is to be connected to T
    If not, click **Create Pool** and make a new pool using any of the available disks.
 
 2. Go to the **Network** page and look at the **Interfaces** card.
-   
-   a. Ensure two interfaces are available and note which is the primary interface that allows SCALE web interface access and access between SCALE systems, TrueCommand, and Active Directory environments.
-      Having two interfaces allows connecting the SCALE systems to Active Directory and using TrueCommand to create and manage the cluster.
 
-   b. Ensure the second interface has a static IP address on a different network/subnet that connects all the SCALE systems.
-      This interface securely handles all the data-sharing traffic between the clustered systems.
+   <ol type="a">
+   <li>Ensure two interfaces are available and note which is the primary interface that allows SCALE web interface access and access between SCALE systems, TrueCommand, and Active Directory environments. Having two interfaces allows connecting the SCALE systems to Active Directory and using TrueCommand to create and manage the cluster.</li>
+   <li>Ensure the second interface has a static IP address on a different network/subnet that connects all the SCALE systems. This interface securely handles all the data-sharing traffic between the clustered systems.</li>
```

Comment:
Maybe the action is to verify and it is assigned a static IP, and is this done in SCALE for the interface or in TrueCommand?  I just noticed your HTML code indicate OL for ordered list. Any reason why you switched to HTML here? Just curious.